frontend-react/src/utils/PermissionsUtils.ts

```diff
@@ -0,0 +1,34 @@
+import { AccessToken } from "@okta/okta-auth-js";
+
+import { getOktaGroups, RSOrgType } from "./OrganizationUtils";
+
+enum PERMISSIONS {
+    /* Non-Okta pseudo groups */
+    SENDER = "DHSender",
+    RECEIVER = "DHReceiver",
+
+    /* Okta groups */
+    PRIME_ADMIN = "DHPrimeAdmins",
+}
+
+const isSender = (s: string) => s.toLowerCase().includes(RSOrgType.SENDER);
+const isAdmin = (s: string) => s.toLowerCase().includes(RSOrgType.ADMIN);
+const isReceiver = (s: string) => !isSender(s) && !isAdmin(s);
+
+const permissionCheck = (
+    level: PERMISSIONS,
+    token: AccessToken | undefined
+): boolean => {
+    if (!token) return false;
+    const oktaGroups = getOktaGroups(token);
+    switch (level) {
+        case PERMISSIONS.SENDER:
```

Comment:
Leave a comment to explain that this'll take into account all okta group memberships, so even if you're in `DHxx_phd`, if you are ALSO in `DHSender_xx-phd`, you'll have access to both sender AND receiver permissions.